src/Filter/NumberFilter.php

```diff
@@ -62,11 +62,9 @@ public function getRenderSettings()
     }
 
     /**
-     * @param string $type
-     *
-     * @return bool
+     * @return string|false
      */
-    private function getOperator($type)
+    private function getOperator(int $type)
     {
         return self::CHOICES[$type] ?? false;
```

Comment:
isnt it better if this returns the default operator instead of false?

Reply:
Indeed. Updated.